### ISSUE TITLE
[diplopia/polyplopia] choose only one name (Trac #1098)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1092]
Get this project over its identity crisis and pick a single name.  Or kill it.  That works too.

Migrated from https://code.icecube.wisc.edu/ticket/1098
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "Get this project over its identity crisis and pick a single name.  Or kill it.  That works too.", 
    "reporter": "david.schultz", 
    "cc": "juancarlos", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[diplopia/polyplopia] choose only one name", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-06T22:12:33", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:09:54: olivas changed milestone from "IceSim 4.3" to "IceSim 5.1"

**Comment by icecube-trac** [comment]
Trac update at 20160802T15:19:42: 
* olivas changed owner from "olivas" to "juancarlos"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160803T14:50:04: juancarlos commented:

> The project's name is legacy from when all it did was combine single events into double events. The name of the I3Module reflects the fact that it combines multiple showers. If this is really a problem, I can rename the project to match the module.

**Comment by icecube-trac** [comment]
Trac update at 20160803T15:00:51: david.schultz commented:

> This is mostly a problem for new people.  Unfortunately, we have a lot of those every year.  So I think it would be best to rename this at some point.  Part of our mission to clean up legacy parts of the code base.

**Comment by icecube-trac** [comment]
Trac update at 20160803T16:22:36: 
* juancarlos changed resolution from "" to "fixed"
* juancarlos commented:

> All references to diplopia have been replaced with polyplopia. See commits 149035-149037. diploplia project replaced by polyplopia.


* juancarlos changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160803T17:18:49: david.schultz commented:

> A few more references:
> 
> ```text
> polyplopia/public/polyplopia/PolyplopiaUtils.h:#ifndef MP_DIPLOPIA_UTILS_H_INLCUDED
> polyplopia/public/polyplopia/PolyplopiaUtils.h:#define MP_DIPLOPIA_UTILS_H_INLCUDED
> polyplopia/public/polyplopia/CoincidentI3ReaderService.h:#ifndef DIPLOPIA_COINCIDENT_I3READER_SERVICE_H_INLCUDED
> polyplopia/public/polyplopia/CoincidentI3ReaderService.h:#define DIPLOPIA_COINCIDENT_I3READER_SERVICE_H_INLCUDED
> polyplopia/public/polyplopia/CoincidentI3ReaderService.h:#endif //DIPLOPIA_COINCIDENT_I3READER_SERVICE_H_INLCUDED
> polyplopia/public/polyplopia/PoissonMerger.h:#ifndef DIPLOPIA_POISSON_H_INLCUDED
> polyplopia/public/polyplopia/PoissonMerger.h:#define DIPLOPIA_POISSON_H_INLCUDED
> polyplopia/private/test/MPHitFilterTest.cxx:TEST_GROUP(I3Diplopia);
> sanity-checker/resources/scripts/simulation_validation/coincident_polygonato.py:from icecube import icetray, phys_services, dataio, diplopia, dataclasses
> sanity-checker/resources/scripts/simulation_validation/coincident_polygonato.py:from icecube.diplopia import CoincidentI3ReaderService
> sanity-checker/resources/scripts/simulation_validation/coincident_polygonato.py:background = diplopia.CoincidentI3ReaderService()
> ```
> 
> There are also several references to DiplopiaWeight floating around, especially in `dst`.
>

**Comment by icecube-trac** [comment]
Trac update at 20160812T15:12:30: 
* david.schultz changed status from "closed" to "reopened"
* david.schultz changed resolution from "fixed" to ""
* david.schultz commented:

> need to move the metaproject externals from `diplopia` project to `polyplopia`

**Comment by icecube-trac** [comment]
Trac update at 20160812T15:12:47: 
* david.schultz changed status from "reopened" to "assigned"
* david.schultz changed owner from "juancarlos" to "olivas"
* david.schultz changed cc from "" to "juancarlos"

**Comment by icecube-trac** [comment]
Trac update at 20160812T15:46:16: 
* olivas changed status from "assigned" to "closed"
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 149214:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149214"
> switch from diplopia to polyplopia.  closes #1092.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas changed milestone from "IceSim 5.1" to ""
* olivas commented:

> Milestone IceSim 5.1 deleted
